gtk/upstream/gtk+3.0/libhdy-sync.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+# -*- coding: UTF-8 -*-
+## Helper script to sync Adwaita from upstream to a destination folder
+## usage:
+##
+##      adwaita-sync.sh --destination <path> [--assets]
```

Comment:
Not really important, but these copy/pasted comments are not accurate here.

Reply:
Fixed in 40c4dde